src/lib/orm/migrations/1594757329224-V13_MigrateAnalytics.ts

```diff
@@ -0,0 +1,120 @@
+import { InfluxDB, Point, WritePrecision } from '@influxdata/influxdb-client';
```

Comment:
Have you locally tested the migration? If not please do so, you can get into a pre-migration state with the `typeorm:schema:resync` script seeing as how this PR does not yet remove the entity or any calls to it.

Reply:
Well, it seems to have worked now.